File: JAVA/P001/src/Use_array.java

public class Use_array {

	public static void main(String[] args) {
		int num[] = {70, 60, 50, 80, 90};
		int min = 9999;
		int n;
		for(n = 0; n < num.length; n++) {
			if(min > num[n]) {
				min = num[n];
			}
		}
		System.out.print("배열 num의 원소들 중에서 최솟값은 ");
		System.out.println(min + "입니다.");
	}

}

File: C/P001/Use_array.c
#include <stdio.h>
void main() {
	int NUM[5] = { 70, 60, 50, 80, 90 };
	int MIN = 9999;
	int N;

	for (N = 0; N < 5; N++) {
		if (MIN > NUM[N]) {
			MIN = NUM[N];
		}
	}
	printf("배열 NUM의 원소들 중에서 최솟값은 %d이다\n", MIN);
	system("pause");
}